packages/core/src/app/ui/dropdown/DropdownTrigger.tsx

```diff
@@ -96,7 +96,7 @@ export default class DropdownTrigger extends Component<DropdownTriggerProps, Dro
         }
 
         this.setState({ shouldShow: false }, () => {
-            document.removeEventListener('click', this.handleClose);
+            document.getElementById('checkout-app')?.removeEventListener('click', this.handleClose);
```

Comment:
Should we move this is in a constant, and put it in a file?

Reply:
I have replaced all occurrences of `checkout-app` with a constant named `CHECKOUT_ROOT_NODE_ID`, and stored it in `packages/core/src/app/checkout/CheckoutRootNodeId.ts`.